Frontend/src/ui/messages/flash.module.scss

```diff
@@ -1,5 +1,12 @@
 @import '../../style/font-sizes';
 
+@media screen and (max-width: 400px) {
```

Comment:
Not a big fan of introducing our own hard-coded breakpoints.
Is the `400px` documented somewhere, or can you reuse breakpoints from SemUI?

Reply:
sure, I'll find them out